src/epcpm/importexportdialog.ui

```diff
@@ -101,40 +101,40 @@
     </widget>
    </item>
    <item row="4" column="0">
-    <widget class="QLabel" name="sunspec_tables_c_label">
+    <widget class="QLabel" name="bitfields_c_label">
      <property name="text">
-      <string>SunSpec Tables C:</string>
+      <string>Bitfields C:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
-    <widget class="QLineEdit" name="sunspec_tables_c"/>
+    <widget class="QLineEdit" name="bitfields_c"/>
    </item>
    <item row="4" column="2">
-    <widget class="QPushButton" name="pick_sunspec_tables_c">
+    <widget class="QPushButton" name="pick_bitfields_c">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
-    <widget class="QLabel" name="sunspec_bitfields_c_label">
+    <widget class="QLabel" name="sunspec_tables_c_label">
      <property name="text">
-      <string>SunSpec Bitfields C</string>
+      <string>SunSpec Tables C:</string>
```

Comment:
should the earlier version have had a ":" also?  looks like that is consistent with the other code.

Reply:
Yes. All the other labels end with a colon, but for some reason this one didn't have it, so I added it in to be consistent.